modules/cnv-cloning-local-volume-to-another-node.adoc

```diff
@@ -0,0 +1,125 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_virtual_machines/cnv_virtual_disks/cnv-moving-local-vm-disk-to-different-node.adoc
+
+[id="cnv-cloning-local-volume-to-another-node_{context}"]
+= Cloning a local volume to another node
+
+You can move a virtual machine disk so that it runs on a specific node by cloning the underlying PersistentVolumeClaim (PVC).
+
+To ensure the virtual machine disk is cloned to the correct node, either create a PersistentVolume (PV) or identify one already on the node and apply a unique label.
+Reference this label in the DataVolume so that the underlying PVC of the virtual machine is cloned to the PV on the correct node. 
+
+[NOTE]
+====
+The destination PV must be the same size or larger than the source PVC or else the cloning operation will fail.
+====
+
+.Prerequisites
+
+* The virtual machine must not be running. Power down the virtual machine before cloning the virtual machine disk. 
+
+.Procedure
+
+. Either create a new local PV on the node, or identify a local PV already on the node:
+
+* Create a local PV that includes the `nodeAffinity.nodeSelectorTerms` parameters. The following manifest creates a 10Gb local PV on node01. 
```

Comment:
Should this be 10Gi (as it's listed in the YAML)?  I would also consider putting the values that you take from the YAML in backticks here.